server/core/templates/login_mech_choose.html

```diff
@@ -11,6 +11,7 @@
 			<form id="login" action="/ui/login/mech_choose" method="post">
 				<input type="hidden" id="mech" name="mech" value="(( mech.value ))" />
 				<button
+					(% if mech.autofocus %)autofocus=true(% endif %)
```

Comment:
This doesn't need the `=true` [per the usual ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus)